src/main/java/mate/jdbc/Main.java

```diff
@@ -1,7 +1,76 @@
 package mate.jdbc;
 
+import mate.jdbc.lib.Injector;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.service.DriverService;
+import mate.jdbc.service.ManufacturerService;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.jdbc");
+
     public static void main(String[] args) {
-        // test your code here
+        ManufacturerService manufacturerService =
+                (ManufacturerService) injector.getInstance(ManufacturerService.class);
+        createManufacturers(manufacturerService);
+
+        DriverService driverService =
+                (DriverService) injector.getInstance(DriverService.class);
+        createDrivers(driverService);
+        System.out.println(manufacturerService.delete(15L));
+        System.out.println(driverService.delete(3L));
+        Driver driver = driverService.get(4L);
+        System.out.println(driver);
+        if (driver != null) {
+            driver.setLicenseNumber("777");
+            System.out.println(driverService.update(driver));
+        }
+        System.out.println(System.lineSeparator());
+        manufacturerService.getAll().forEach(System.out::println);
+        System.out.println(System.lineSeparator());
+        driverService.getAll().forEach(System.out::println);
+    }
+
+    private static void createManufacturers(ManufacturerService manufacturerService) {
+        Manufacturer manufacturerAudi = new Manufacturer();
```

Comment:
Why variable name **manufacturerAudi** if we set BMW? It's confusing a reader a little bit. Maybe better to rename variables in this method?

Reply:
This confuses fixed